.github/workflows/remove-experimental.yml

```
@@ -27,7 +27,7 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         run: |
-          pushd serverless/regulations-site
+          pushd serverless/regulations
```

should just be `serverless`

```suggestion
          pushd serverless
```